libs/types/src/election.ts

```diff
@@ -459,13 +459,20 @@ export interface GridLayout {
   readonly ballotStyleId: BallotStyleId;
   readonly columns: number;
   readonly rows: number;
+  /**
+   * Area in timing mark units around a target mark (i.e. bubble) to consider
+   * part of the option for that target mark. This is used to crop the image
+   * to show the write-in area for a given grid position.
+   */
+  readonly optionBoundsFromTargetMark: Rect;
```

Comment:
How do you think about `gridLayouts` vs `ballotLayout`? Since `ballotLayout.targetMarkPosition` is related to the option bounds, I could see a case for colocating them

Reply:
One potential advantage of putting this in `ballotLayout` is reduced duplication. However, it also precludes having e.g. an election with both state and local ballots that have different values. Maybe that wouldn't ever happen, though. The biggest reason I put it here is because if I put it in `ballotLayout` it has to be optional, but I wanted it to be required if `gridLayouts` is present. We could enforce that with a Zod `refine` call, but I preferred to let the type system handle it.

Reply:
Ah, yeah I haven't thought about elections that might have multiple entries in `gridLayouts` before...

What is `ballotLayout` used for? Should it potentially be moved inside `gridLayouts`?

Maybe all of this is moot if we switch to CDF for this anyway. We don't need to over-rotate on it now

Reply:
I could maybe see us merging them together, but I agree we don't need to fix this now.